src/libstore/builtins/buildenv.hh

```diff
@@ -12,6 +12,40 @@ struct Package {
     Package(const Path & path, bool active, int priority) : path{path}, active{active}, priority{priority} {}
 };
 
+class BuildProfileConflictError : public Error
+{
+public:
+    const Path originalFile;
+    const Path originalPkgPath;
+    const Path conflictingFile;
+    const Path conflictingPkgPath;
+    int priority;
+
+    BuildProfileConflictError(
+        const Path originalFile,
+        const Path originalPkgPath,
+        const Path conflictingFile,
+        const Path conflictingPkgPath,
+        int priority
+    )
```

Comment:
Idem, use designated initializers.

Reply:
By using struct this works. Incorporated into the PR :+1: Thanks for the suggestion, named-parameters-like is much clearer to read.